### DISCUSSION TITLE
project management: replacing dephell

### DISCUSSION THREAD

**Discussion by pwoolvett** [flakeheaven/flakeheaven#19] [Archived]
We should replace dephell. Its archived and outdated, and does not work out of the box (dependencies conflict to start with).

TODO

*  [x] VOTE! for replacement
*  [ ] Add developer guidelines and documentation to ease support and local testing before submitting pr
   * [x] readme / docs section or page with a step-by-step instruction to setup the project
   * [ ] issue templates
   * [ ] pr templates

**Comment by pwoolvett** [comment]
Poetry

**Reply by jose-rojas0** [reply]
If we go this route I'd be willing to take this on 👍

**Reply by pwoolvett** [reply]
sure @jarojasm95, go ahead with the PR

In upcoming poetry release we'll have dependency groups, I think for the moment we could just include dev,docs,etc as extras

I think this would be a good opportunity to pin the deps, so we avoid dependency hell (!) all over again.

**Reply by pwoolvett** [reply]
@jarojasm95 ping

**Reply by jose-rojas0** [reply]
https://github.com/flakeheaven/flakeheaven/pull/24

**Comment by pwoolvett** [comment]
pipenv

**Comment by pwoolvett** [comment]
flit

**Reply by pwoolvett** [reply]
pro: pyproject.toml already using it

**Comment by pwoolvett** [comment]
plain setup.py

**Reply by pwoolvett** [reply]
pro: no additional requirements on the dev side

**Comment by pwoolvett** [comment]
solved at #24
